eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -113,8 +113,8 @@ jobs:
         displayName: 'Build and Package'
         inputs:
           mavenPomFile: pom.xml
-          goals: 'package'
-          options: '$(DefaultOptions) "-DpackageOutputDirectory=$(Build.ArtifactStagingDirectory)" -DskipTests -Dinject-codesnippets -Dgenerate-overview -pl $(ProjectList) -am' # We include template-module so we ensure it always builds in CI
```

Comment:
We probably should also remove the packageOutputDirectory property and the ant copy step after we make this switch to deploy.

Reply:
@weshaggard do we really want to remove that? I know when I'm building locally I typically set the packageOutputDirectory on my command line, it's more of an ease of use thing.

Reply:
If we want deploy to the be the main goal for producing output then maybe we do want to remove it.

Reply:
I know for me it was one of those things that confused me when I first started working with Maven in depth on these team. I assumed that it was something that Maven did automatically until I realized we had this bit of Ant magic in our parent POM.

I'm all for magic if it is truly useful, but given Maven provides an OOB solution for spitting packages into a repository I'm not sure its that useful, and is possibly confusing.